### PR TITLE
Add database seeds

### PR DESCRIPTION
**Pull request by drewpotter**
Opening a PR so that @spencerldixon can see the code as I develop this branch further and make comments if necessary.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -16,3 +16,199 @@
...
+    text: "The best way for conservation to contribute to human wellbeing is by promoting economic growth",
+  },
+  {
+    text: "When communities manage their own resources, their efforts are more effective than top-down approaches",
+  }
+]
+
+questions.each do |question|
+  Question.where(text: question[:text]).first_or_create do |q|
+    q.text = question[:text]
```
Comment (spencerldixon) [inline db/seeds.rb]: Since we only have text, this line is unnecessary, we can probably simplify this statement a bit

File: db/seeds.rb [hunk 2]
```diff
@@ -16,3 +16,199 @@
...
+    text: "If you are answering this as part of the WWF UK survey please choose your programme",
+  },
+  {
+    text: "Are you taking part in the Ol Pejeta staff survey?",
+  }
+]
+
+demographic_questions.each do |demographic_question|
+  DemographicQuestion.where(text: demographic_question[:text]).first_or_create do |dq|
+    dq.text = demographic_question[:text]
```
Comment (spencerldixon) [inline db/seeds.rb]: Same deal here

File: db/seeds.rb (hunk 1 above)
Comment (drewpotter) [reply db/seeds.rb]: Thanks @spencerldixon I will do that!

File: db/seeds.rb (hunk 2 above)
Comment (drewpotter) [reply db/seeds.rb]: Same I will do that too!

**Pull request merged**